docs/source/developers/cpp/windows.rst

```diff
@@ -214,7 +288,7 @@ Debug builds
 ============
 
 To build a Debug version of Arrow, you should have pre-installed a Debug
-version of Boost. It's recommended to configure cmake build with the
+version of Boost. It's recommended to configure ``cmake build`` with the
```

Comment:
It's not `cmake build` that you are configuring, it's `cmake`.

Reply:
Fixed in 945642a8b6bc59c7b384ef17fefe2af76229de3e